### ISSUE TITLE
How to initialize unremovable tags (read-only)?

### ISSUE THREAD

**Issue by nandude** [alexgorbatchev/jquery-textext#162]
My application has to pages which uses Textext: one is for saving tags, and one is to show read-only tags. Textext is initialized to a text input. The problem is that I couldn't manage to create read-only  (so there is no X to remove a tag) tags yet. First I tried to set the text input disabled ( in jquery: $('.input').prop('disabled', 'disabled'); ) which of course showed the remover X. In Chrome I couldn't click on the X, but in Firefox I could. 

To sum it up: is there any way, to create read-only tags with the plugin, or easily set tags unclickable in any browser?

**Comment by shri1920** [comment]
Hi,

Try to hide the "x" after initializing the plugin.

$(".text-remove").hide() will help you to hide the "x".
